charts/fybrik/files/taxonomy/data_catalog_request.json

```diff
@@ -0,0 +1,17 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Data Catalog Request - GetDatasetDetails",
+	"type": "object",
+	"properties": {
+	  "assetID": {
+		"type": "string"
+	  },
+	  "operationType": {
+		"$ref": "taxonomy.json#/definitions/OperationType"
+	  }
+	},
+	"additionalProperties": false,
+	"required": [
+	  "assetID"
+	]
+}
```

Comment:
Please fix formatting on all files (missing empty line at end of file)

Reply:
done formatting with vscode. However it does not insert empty blank line at the end of file. Hope it is ok. Thanks.

Reply:
Consider setting up your IDE with a better formatter as you also have inconsistencies in space indentation.

Reply:
changed the formatting. Please review. Thanks.